package.json

```diff
@@ -9,6 +9,7 @@
     "babel-preset-es2015": "^6.22.0",
     "babel-preset-react": "^6.22.0",
     "babelify": "^7.3.0",
+    "decimal.js-light": "^2.2.0",
```

Comment:
I don't think we need to force a precision during intermediate computation. When we output to the browser we round to two decimal places anyway.